rust_src/src/windows.rs

```diff
@@ -1074,6 +1074,42 @@ pub fn window_new_total(window: LispWindowValidOrSelected) -> LispObject {
     win.new_total
 }
 
+/// Setter for new total of Window
+#[no_mangle]
+pub extern "C" fn wset_new_total(win: &mut LispWindowRef, new_total: EmacsInt) {
+    win.new_total = new_total.into();
+}
+
+/// Set new total size of WINDOW to SIZE.
+/// WINDOW must be a valid window and defaults to the selected one.
+/// Return SIZE.
+///
+/// Optional argument ADD non-nil means add SIZE to the new total size of
+/// WINDOW and return the sum.
+///
+/// The new total size of WINDOW, if valid, will be shortly installed as
+/// WINDOW's total height (see `window-total-height') or total width (see
+/// `window-total-width').
+///
+/// Note: This function does not operate on any child windows of WINDOW.
+#[lisp_fn(min = "2")]
+pub fn set_window_new_total(
+    window: LispWindowValidOrSelected,
+    size: EmacsInt,
+    add: LispObject,
+) -> LispObject {
+    let win: LispWindowRef = window.into();
+
+    let new_total: EmacsInt = if add.is_nil() {
+        size
+    } else {
+        EmacsInt::from(win.new_total) + size
+    };
+    wset_new_total(&mut win, new_total);
```

Comment:
You can directly assign `win.new_total` here.
I don't see any advantage in porting setter method `wset_new_total` as it will be removed from `C` code base once all of its callers are ported.

Reply:
OK. that's right. 
However, I don't understand why the original C code uses setter instead of direct assignment. 
Anyway, I will directly assign win.new_total.

Reply:
https://debbugs.gnu.org/cgi/bugreport.cgi?bug=12215